download.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/zmk.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ZMK Studio - Download</title>
```

Comment:
**thought:** not required but it'd be nice to have when sharing the app link.

 ```suggestion
<title>Download ZMK Studio</title>
<meta name="description" content="ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards.">
        
<meta property="og:title" content="Download ZMK Studio">
<meta property="og:description" content="ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards.">
<meta property="og:image" content="https://zmk.studio/zmk.svg">
<meta property="og:url" content="https://zmk.studio/download">
<meta property="og:site_name" content="Download ZMK Studio">
<meta property="og:type" content="website">
        
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:title" content="Download ZMK Studio">
<meta name="twitter:description" content="ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards.">
<meta name="twitter:image" content="https://zmk.studio/zmk.svg">
```

Reply:
Fair, I think we should do it as a follow-up since index.html is lacking this as well.